ci/circleci/README.md

```diff
@@ -0,0 +1,16 @@
+<p align="center">
+  <img src="https://secrethub.io/img/integrations/circleci/github-banner.png?v1" alt="CircleCI + SecretHub" width="390">
+</p>
+<br/>
+
+<p align="center">
+  <a href="https://secrethub.io/integrations/circleci/"><img alt="Learn More" src="https://secrethub.io/img/buttons/github/learn-more.png?v1" height="28" /></a>
+  <a href="https://secrethub.io/docs/guides/circleci/"><img alt="View Docs" src="https://secrethub.io/img/buttons/github/view-docs.png?v1" height="28" /></a>
+  <a href="https://github.com/secrethub/secrethub-circleci-orb"><img alt="View Source Code" src="https://secrethub.io/img/buttons/github/view-source.png?v1" height="28" /></a>
+  <a href="https://circleci.com/orbs/registry/orb/secrethub/cli"><img alt="View Orb" src="https://secrethub.io/img/buttons/github/view-orb.png?v1" height="28" /></a>
```

Comment:
These images still have to be created and published.

Reply:
Not a fan of 4 buttons next to each other. Is not clear to the user what to pick. And what color should they be?

My suggestion: since the first two buttons (integration page and guide) are more important IMO than the ones about the orb, I'd say let's skip those.

Reply:
Yes sounds good.

Do you still want to link to the other two places in the readme or do you think it's enough to have those indirectly accessible through the `learn more` button?

`Learn more` => `View In Registry`
and
`Learn more` => `View In Registry` => `Source`

Reply:
I think we've got it covered with just the two buttons.

Less is more.